main.js

```diff
@@ -65,8 +65,9 @@ export function statusTypeMatches({credential} = {}) {
   if(!Array.isArray(contexts)) {
     throw new TypeError('"@context" must be an array.');
   }
-  if(contexts[0] !== CONTEXTS.VC_V1) {
-    throw new Error(`The first "@context" value must be "${CONTEXTS.VC_V1}".`);
+  if(contexts[0] !== VC_V1_CONTEXT_URL) {
+    throw new Error(`The first "@context" value must be ' +
+      '"${VC_V1_CONTEXT_URL}".`);
```

Comment:
This line opens with a backtick and intends to close with a single quote -- but instead erroneously becomes a multiline string. Regardless, it should have been dropped to the next line:

```suggestion
    throw new Error(
      `The first "@context" value must be "${VC_V1_CONTEXT_URL}".`);
```